protocol_tests/connection/__init__.py

```diff
@@ -0,0 +1,236 @@
+import base64
+import re
+import uuid
+
+from voluptuous import Schema, Optional
+from aries_staticagent import Message, crypto
+from ..schema import MessageSchema
+
+
+class DIDDoc(dict):
+    """DIDDoc class for creating and verifying DID Docs."""
+    VALIDATOR = Schema({
+        "@context": "https://w3id.org/did/v1",
+        "id": str,
+        "publicKey": [{
+            "id": str,
+            "type": "Ed25519VerificationKey2018",
```

Comment:
Not sure what the ramifications of this line ("type": "Ed25519...") are. Does it cause the schema validator to only accept DID Docs where the publicKey section must consist entirely of Ed25519 keys?

Reply:
Ah; that is accurate. That's a carry over from previous restrictions/shortsightedness. I'll call this PR good for now but we should address this in future PRs.